pkg/remote/aws/repository/elb_repository.go

```diff
@@ -0,0 +1,46 @@
+package repository
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/aws/aws-sdk-go/service/elb/elbiface"
+	"github.com/snyk/driftctl/pkg/remote/cache"
+)
+
+type ELBRepository interface {
+	ListAllLoadBalancers() ([]*elb.LoadBalancerDescription, error)
+}
+
+type elbRepository struct {
+	client elbiface.ELBAPI
+	cache  cache.Cache
+}
+
+func NewELBRepository(session *session.Session, c cache.Cache) *elbRepository {
+	return &elbRepository{
+		elb.New(session),
+		c,
+	}
+}
+
+func (r *elbRepository) ListAllLoadBalancers() ([]*elb.LoadBalancerDescription, error) {
+	cacheKey := "elbListAllLoadBalancers"
```

Comment:
We're gonna have a problem here since this is the same `cacheKey` than the one in the ELBv2 resource.

Let's change the one in the v2 resource to `elbv2ListAllLoadBalancers` to be inlined with the rest.

Reply:
Very good catch!!